packages/beacon-node/src/chain/blocks/types.ts

```diff
@@ -11,7 +11,7 @@ export enum BlockInputType {
 
 export type BlockInput =
   | {type: BlockInputType.preDeneb; block: allForks.SignedBeaconBlock}
-  | {type: BlockInputType.postDeneb; block: allForks.SignedBeaconBlock; blobs: deneb.BlobsSidecar};
+  | {type: BlockInputType.postDeneb; block: allForks.SignedBeaconBlock; blobs: deneb.BlobsSidecar[]};
```

Comment:
Should this be BlobSidecar[] ?

Also a few other places where BlobsSidecar remains

Reply:
wip :slightly_smiling_face: